tb_plugin/torch_tb_profiler/io/file.py

```diff
@@ -80,6 +80,16 @@ def get_filesystem(filename):
         raise ValueError("No recognized filesystem for prefix %s" % prefix)
     return fs
 
+# When subprocesses are started without state, they won't retain
```

Comment:
what's the reason that need this change? I tried locally by using forkserver start method and there is no need to backup and restore the file systems.

Reply:
When the subprocess is started using `forkserver`, it will retain any filesystems that are added inside of `file.py` because those lie inside of its import paths. So things like Azure, GCP, S3, etc... will still work.

However, if a filesystem is added using `register_filesystem()`, then that filesystem will be lost during the initialization, because the subprocess doesn't retain state from its parent. This change is necessary to retain filesystems which were added dynamically.

Reply:
From the time being, the register_filesystem is only called in file.py. Do you have cases that call register_filesystem? If no, I don't suggest this change.

Reply:
with my change in https://github.com/pytorch/kineto/pull/344, the get_all_filesystems/restore_filesystems is needless.

Reply:
Great, let me test that change and see if it works on my end.

Reply:
@Reubend, please let me know whether my change works or not for your cases.